diff --git a/node_modules/@starport/vue/src/components/SpAmountSelect/SpAmountSelect.vue b/node_modules/@starport/vue/src/components/SpAmountSelect/SpAmountSelect.vue
index b30d2c5..9d26e5f 100644
--- a/node_modules/@starport/vue/src/components/SpAmountSelect/SpAmountSelect.vue
+++ b/node_modules/@starport/vue/src/components/SpAmountSelect/SpAmountSelect.vue
@@ -20,7 +20,7 @@
             error: !hasEnoughBalance(x, x.amount.amount)
           }"
         >
-          {{ getBalanceAmount(x) }} available
+          {{ Intl.NumberFormat('en-GB').format(Number(getBalanceAmount(x))/1000000) }} available
         </div>
       </div>
 
@@ -126,7 +126,7 @@
                 </div>
 
                 <div class="token-amount">
-                  {{ parseAmount(x.amount.amount) }} available
+                  {{ Intl.NumberFormat('en-GB').format(parseAmount(x.amount.amount)/1000000) }} available
                 </div>
               </div>
             </div>
diff --git a/node_modules/@starport/vue/src/components/SpAssets/SpAssets.vue b/node_modules/@starport/vue/src/components/SpAssets/SpAssets.vue
index 738e112..e2a3161 100644
--- a/node_modules/@starport/vue/src/components/SpAssets/SpAssets.vue
+++ b/node_modules/@starport/vue/src/components/SpAssets/SpAssets.vue
@@ -79,10 +79,10 @@
         >
           <td class="assets-table__denom">
             <div class="sp-denom-marker">
-              <SpDenom :denom="balance.amount.denom" modifier="avatar" />
+              <SpDenom :denom="balance.amount.denom == 'ufetf' ? 'fetf' : balance.amount.denom" modifier="avatar" />
             </div>
             <div class="sp-denom-name">
-              <SpDenom :denom="balance.amount.denom" />
+              <SpDenom :denom="balance.amount.denom == 'ufetf' ? 'fetf' : balance.amount.denom" />
             </div>
           </td>
           <td
@@ -103,7 +103,7 @@
             </ul>
           </td>
           <td class="assets-table__amount">
-            {{ new Intl.NumberFormat('en-GB').format(balance.amount.amount) }}
+            {{ new Intl.NumberFormat('en-GB').format(balance.amount.amount/1000000) }}
           </td>
         </tr>
         <tr v-if="noSearchResults" class="assets-table__row">
diff --git a/node_modules/@starport/vue/src/components/SpDenom/SpDenom.vue b/node_modules/@starport/vue/src/components/SpDenom/SpDenom.vue
index 1881139..b0976f2 100644
--- a/node_modules/@starport/vue/src/components/SpDenom/SpDenom.vue
+++ b/node_modules/@starport/vue/src/components/SpDenom/SpDenom.vue
@@ -47,7 +47,9 @@ export default defineComponent({
       }
     })
 
-    return { sizeClassObject }
+    const denom = "ufetf" ? "fetf" : props.denom
+
+    return { denom, sizeClassObject }
   }
 })
 </script>
diff --git a/node_modules/@starport/vue/src/composables/useKeplr.ts b/node_modules/@starport/vue/src/composables/useKeplr.ts
index 4fd0968..0d5153a 100644
--- a/node_modules/@starport/vue/src/composables/useKeplr.ts
+++ b/node_modules/@starport/vue/src/composables/useKeplr.ts
@@ -35,9 +35,9 @@ export default function ({ $s }: Params): Response {
       let addrPrefix = $s.getters['common/env/addrPrefix']
 
       let stakeCurrency = {
-        coinDenom: staking.params.bond_denom.toUpperCase(),
-        coinMinimalDenom: staking.params.bond_denom,
-        coinDecimals: 0
+        coinDenom: "FETF",
+        coinMinimalDenom: "ufetf",
+        coinDecimals: 6
       }
 
       let bip44 = {
@@ -56,28 +56,28 @@ export default function ({ $s }: Params): Response {
       let currencies = tokens.supply.map((x: Amount) => {
         const y: AmountWithMeta = {
           amount: '0',
-          denom: '',
-          coinDenom: '',
-          coinMinimalDenom: '',
-          coinDecimals: 0
+          denom: 'FETF',
+          coinDenom: 'FETF',
+          coinMinimalDenom: 'ufetf',
+          coinDecimals: 6
         }
-        y.coinDenom = x.denom.toUpperCase()
-        y.coinMinimalDenom = x.denom
-        y.coinDecimals = 0
+        y.coinDenom = 'FETF'
+        y.coinMinimalDenom = 'ufetf'
+        y.coinDecimals = 6
         return y
       })
 
       let feeCurrencies = tokens.supply.map((x: Amount) => {
         const y: AmountWithMeta = {
           amount: '0',
-          denom: '',
-          coinDenom: '',
-          coinMinimalDenom: '',
-          coinDecimals: 0
+          denom: 'FETF',
+          coinDenom: 'FETF',
+          coinMinimalDenom: 'ufetf',
+          coinDecimals: 6
         }
-        y.coinDenom = x.denom.toUpperCase()
-        y.coinMinimalDenom = x.denom
-        y.coinDecimals = 0
+        y.coinDenom = 'FETF'
+        y.coinMinimalDenom = 'ufetf'
+        y.coinDecimals = 6
         return y
       })
 
@@ -89,6 +89,8 @@ export default function ({ $s }: Params): Response {
         high: 0.04
       }
 
+      chainName = "Defund"
+
       if (chainId) {
         await window.keplr.experimentalSuggestChain({
           features,
@@ -107,8 +109,8 @@ export default function ({ $s }: Params): Response {
 
         window.keplr.defaultOptions = {
           sign: {
-            preferNoSetFee: true,
-            preferNoSetMemo: true
+            preferNoSetFee: false,
+            preferNoSetMemo: false
           }
         }
 
